packages/spor-react/src/alert/ServiceAlert.tsx

```diff
@@ -56,9 +58,9 @@ export const ServiceAlert = ({
   onToggle = () => {},
   ...boxProps
 }: ServiceAlertProps) => {
-  variant = "service";
   const { t } = useTranslation();
-  const styles = useMultiStyleConfig("AlertService");
+  const styles = useMultiStyleConfig("AlertService", { variant });
+  let isGlobalDeviation = variant === "global-deviation" ? "black" : "white";
```

Comment:
Siden denne nå er en farge, så burde ikke navnet tyde på en bool! Enten gjør om variabelen til en bool eller endre navn. Burde forøvrig også være en `const` 😊 Og er du sikker på at fargen skal være ren sort? Tror vi pleier å bruke `darkGrey` på all mørk tekst i Spor, og lurer på om ikoner også har samme farge (men sjekk det med designere)

Reply:
Forøvrig; trenger du denne? Burde du ikke bruke fargene du har definert i `alert-service.ts` filen som du henter ut i `styles` her?

Reply:
Enig med Cecilie, denne kan flyttes til container i styling-filen